app/components/InfoOverlay/Overlay.js

```diff
@@ -65,7 +65,7 @@ function Overlay({
         <LayerContent flex={{ grow: 1 }}>
           <div>
             {markdown && (
-              <Markdown source={content} className="react-markdown" linkTarget="_blank" />
```

Comment:
@vbojilova what about the linkTarget option?

Reply:
it causes this error: "react-dom.development.js:26957 Uncaught Assertion: Unexpected `linkTarget` prop, remove it"

Reply:
ok but we should not change the functionality ... if it doesnt by default, I am sure there is an alternative way of achieving this

Reply:
see https://github.com/remarkjs/react-markdown/pull/762

Reply:
https://github.com/remarkjs/react-markdown/pull/762#issuecomment-1806576871

Reply:
Ha! Good point! I re-added the functionality back in here [c5d736c](https://github.com/impactoss/impactoss-client/pull/558/commits/c5d736c3af28e4e76ff019010e48957da08bfd50)